standStillHeadway: 12.0  # Standstill gap between vehicles (m)
maxAccel: 1.5 # Maximum acceleration absolute value used in controller filters (m/s^2)
Kp: 0.4 # Proportional weight for PID controller
Kd: 0.05 # Derivative Weight for PID controller
Ki: 0.0 # Integral weight for PID controller
maxValue: 2.0 # Max value to restrict speed adjustment at one time step (limit on delta_v) (m/s)
minValue: -10.0 # Min value to restrict speed adjustment at one time step (limit on delta_v) (m/s)
cmdTmestamp: 100  # Timestep to calculate ctrl commands (ms)
adjustmentCap: 11.0 # Adjustment cap for speed command (m/s)
dt: 0.1 # Timestep to calculate ctrl commands (s)
integratorMax: 100 # Max limit for integrator term
integratorMin: -100 # Min limit for integrator term
lowpassGain: 1.0  #Lowpass filter gain
lookaheadRatio: 1.6  # Ratio to calculate lookahead distance
minLookaheadDist: 15.0  # Min lookahead distance (m)
vehicleLength: 5.0 # Average length of a vehicle (m)
ss_theta: 1.5 # Minimum speed to be considered as moving (m/s)
standstill: 2.0 # Extra time needed to reacte to traffic sceanrios when vehicle is standstill (not moving) (s).
inter_tau: 1.5 # Inter-platoon time gap, refer to bumper to bumper gap time (s)
intra_tau: 0.6  #Intra-platoon time gao, refer to bumper to bumper gap time (s)
gap_weight: 0.9 # Weighted ratio for time-gap based calculation,
time_step: 5  # time step ga regulation algorithm uses to calculate desired position (s).
test_front_join: false  #Flag to enable/disable front join functionality with two vehicles.
